Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'dynflow', '>= 1.6.5', '< 2.0.0'
 gem 'daemons'
 gem 'bcrypt', '~> 3.1'
 gem 'get_process_mem'
-gem 'rack-cors', '~> 1.0.2', require: 'rack/cors'
+gem 'rack-cors', '~> 1.1', require: 'rack/cors'
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/8428

Reply:
Packaging PR is approved, feel free to merge them as you wish.